recipes/itk/all/patches/0001-find-conan-packages.patch

```diff
@@ -0,0 +1,157 @@
+ endif()
+@@ -52,9 +52,9 @@
+ 
+ if(ITK_USE_SYSTEM_EIGEN)
+   set(_Eigen3_SYSTEM_OR_INTERNAL "Eigen3")
+-  find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED CONFIG)
+-  set(Eigen3_DIR_INSTALL ${Eigen3_DIR})
+-  set(Eigen3_DIR_BUILD ${Eigen3_DIR})
++  find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED)
++  set(Eigen3_DIR_INSTALL ${Eigen3_INCLUDE_DIR})
++  set(Eigen3_DIR_BUILD ${Eigen3_INCLUDE_DIR})
+ else()
+   set(_Eigen3_SYSTEM_OR_INTERNAL "ITKInternalEigen3")
+   find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED CONFIG)
+@@ -66,7 +66,7 @@
+ # Eigen3 is header only, but there are compile definitions that we want to provide
+ # to enforce use of MPL only code, and to disable warnings.
+ # We only need to add the location of the header itk_eigen.h used internally.
+-get_target_property(Eigen_INCLUDE_DIRS ${_eigen_itk_target} INTERFACE_INCLUDE_DIRECTORIES)
++set(Eigen_INCLUDE_DIRS ${Eigen3_INCLUDE_DIRS})
+ set(ITKEigen3_INCLUDE_DIRS
+   ${Eigen_INCLUDE_DIRS}
+   ${ITKEigen3_BINARY_DIR}/src # For the generated itk_eigen.h
+@@ -75,14 +75,10 @@
+ # When this module is loaded by an app, load Eigen too.
+ # Load ITKInternalEigen3 or Eigen3 depending on ITK_USE_SYSTEM_EIGEN
+ set(ITKEigen3_EXPORT_CODE_INSTALL "
+-set(ITK_USE_SYSTEM_EIGEN \"${ITK_USE_SYSTEM_EIGEN}\")
+-set(${_Eigen3_SYSTEM_OR_INTERNAL}_DIR \"${Eigen3_DIR_INSTALL}\")
+-find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED CONFIG)
++find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED)
+ ")
+ set(ITKEigen3_EXPORT_CODE_BUILD "
+-set(ITK_USE_SYSTEM_EIGEN \"${ITK_USE_SYSTEM_EIGEN}\")
+-set(${_Eigen3_SYSTEM_OR_INTERNAL}_DIR \"${Eigen3_DIR_BUILD}\")
+-find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED CONFIG)
++find_package(${_Eigen3_SYSTEM_OR_INTERNAL} ${_Eigen3_min_version} REQUIRED)
+ ")
+ 
+ # Eigen3 targets are not installed if ITK_USE_SYSTEM_EIGEN==True
+--- Modules/ThirdParty/Eigen3/src/itk_eigen.h.in
++++ Modules/ThirdParty/Eigen3/src/itk_eigen.h.in
+@@ -43,10 +43,10 @@
+   set(Eigen3_DIR ${ITKInternalEigen3_DIR})
+ endif()
+ message(STATUS "From ITK: Eigen3_DIR: ${Eigen3_DIR}")
+-find_package(Eigen3 REQUIRED CONFIG)
++find_package(Eigen3 REQUIRED)
+ add_executable(main main.cpp)
+ target_link_libraries(main PUBLIC ${ITK_LIBRARIES})
+-target_link_libraries(main PUBLIC Eigen3::Eigen)
++target_link_libraries(main PUBLIC Eigen3::Eigen3)
+ \endcode
+  *
+  * Then, the user can use Eigen3 as usual:
+--- Modules/ThirdParty/JPEG/CMakeLists.txt
++++ Modules/ThirdParty/JPEG/CMakeLists.txt
+@@ -6,13 +6,13 @@
+ 
+ if(ITK_USE_SYSTEM_JPEG)
+-  find_package(JPEG REQUIRED)
++  find_package(OpenJPEG REQUIRED)
+   set(ITKJPEG_INCLUDE_DIRS
+     ${ITKJPEG_BINARY_DIR}/src
+     )
+   set(ITKJPEG_SYSTEM_INCLUDE_DIRS
+-    ${JPEG_INCLUDE_DIR}
++    #(libjpeg_INCLUDE_DIRS}
+     )
+-  set(ITKJPEG_LIBRARIES "${JPEG_LIBRARIES}")
++  set(ITKJPEG_LIBRARIES JPEG::JPEG)
```

Comment:
This part seems weird. Why finding openjpeg instead of libjpeg, and after link libjpeg?

Reply:
Fixed it.
libjpeg is the name of the recipe providing JPEG.
This recipe  is based on #262, which is a old version of cci, that does not set cmake targets/variables correctly.
Thanks for the review.